package.json

```diff
@@ -45,19 +50,23 @@
     "react-apollo-network-status": "^5.1.0",
     "react-chartjs-2": "^3.0.5",
     "react-countup": "^6.0.0",
-    "react-currency-input-field": "^3.5.1",
+    "react-currency-input-field": "^3.6.0",
     "react-dom": "^17.0.2",
     "react-hook-form": "^7.19.5",
     "react-intl": "^5.20.12",
     "react-multi-provider": "^0.1.5",
     "react-router-dom": "^6.0.2",
     "react-scripts": "4.0.3",
+    "react-text-mask": "^5.4.3",
     "react-use": "^17.3.1",
     "save": "^2.4.0",
+    "text-mask-addons": "^3.8.0",
     "typeface-roboto": "^1.1.13",
     "typescript": "^4.1.2",
     "use-count-up": "^3.0.1",
+    "use-hooks": "^2.0.0-rc.5",
```

Comment:
should we protect ouselves a little more with this dep since it's `-rc` ? Hard pin?

Reply:
I guess ^rc will be fine as long as they semver since we'll upgrade patch and feature anyway? The question might rather be shouldn't we always hard pin and upgrade?